app/models/manageiq/providers/openshift/container_manager.rb

```diff
@@ -22,7 +24,26 @@ def self.event_monitor_class
 
   def create_project(project)
     connect.create_project_request(project)
-  rescue KubeException => e
-    raise MiqException::MiqProvisionError, "Unexpected Exception while creating project: #{e}"
+  end
+
+  def users_from_provider
+    connect.get_users
+  end
+
+  def user_from_provider(user_name)
+    connect.get_user(user_name)
+  end
+
+  def user_exists_in_provider?(user_name)
+    !user_from_provider(user_name).nil?
+  end
+
+  def add_user_in_provider(user_name)
+    user = Kubeclient::Resource.new
```

Comment:
@kevensen  you can create the user (and the project request in the spec) like that: https://github.com/ManageIQ/manageiq-providers-kubernetes/blob/master/spec/models/manageiq/providers/kubernetes/container_manager/scanning/job_spec.rb#L190 (you can also skip the variable that way)